routers/auth.py

```diff
@@ -1,15 +1,21 @@
-from fastapi import APIRouter,Response
+from fastapi import APIRouter, Response, Header;
 from email_validator import validate_email,EmailNotValidError
```

Comment:
Please keep the following convention in mind to increase code readability 

`import third party`
space
`import local`